### DISCUSSION TITLE
SHODAN_API_KEY

### DISCUSSION THREAD

**Discussion by imbas007** [projectdiscovery/nuclei#3311] [Q&A] [answered]
![Screenshot_15](https://user-images.githubusercontent.com/100182585/218318156-45532822-b7dd-41ed-bcff-46f95c832723.png)

**Comment by ehsandeep** [comment, marked as answer]
@imbas007 You need to set shodan api key as you can see in the error msg, more details - https://nuclei.projectdiscovery.io/nuclei/get-started/#uncover-integration
